buckets/local/bucket_test.go

```diff
@@ -353,12 +365,24 @@ func TestBucket_Watch(t *testing.T) {
 	go ec.collect(events)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
+	var wg sync.WaitGroup
+	var onlineStateCount, offlineStateCount int
 	go func() {
-		err = buck1.Watch(ctx, WithWatchEvents(events))
-		require.Nil(t, err)
+		wg.Add(1)
```

Comment:
Nit: I'd move this add out of `go func()`, since `wg.Wait()` could happen before the addition. Being realistic, with the current logic that's impossible without things failing... but just a conceptual nit.